test/spec/modules/zeotapIdPlusIdSystem_spec.js

```diff
@@ -0,0 +1,154 @@
+import { expect } from 'chai';
+import find from 'core-js-pure/features/array/find.js';
+import { config } from 'src/config.js';
+import { newStorageManager } from 'src/storageManager.js';
+import { init, requestBidsHook, setSubmoduleRegistry } from 'modules/userId/index.js';
+import { zeotapIdPlusSubmodule } from 'modules/zeotapIdPlusIdSystem.js';
+
+const storage = newStorageManager();
+
+const ZEOTAP_COOKIE_NAME = 'IDP';
+const ZEOTAP_COOKIE = 'THIS-IS-A-DUMMY-COOKIE';
+const CONFIG_PARAMS_MOCK = {
+  name: 'zeotapIdPlus',
+  storage: { name: 'IDP', type: 'cookie' }
+};
+
+function getConfigMock() {
+  return {
+    userSync: {
+      syncDelay: 0,
+      userIds: [{
+        name: 'zeotapIdPlus',
+        storage: {
+          name: 'IDP',
+          type: 'cookie'
+        }
+      }]
+    }
+  }
+}
+
+function getAdUnitMock(code = 'adUnit-code') {
+  return {
+    code,
+    mediaTypes: {banner: {}, native: {}},
+    sizes: [
+      [300, 200],
+      [300, 600]
+    ],
+    bids: [{
+      bidder: 'sampleBidder',
+      params: { placementId: 'banner-only-bidder' }
+    }]
+  };
+}
+
+function writeZeotapCookie() {
+  storage.setCookie(
+    ZEOTAP_COOKIE_NAME,
+    btoa(JSON.stringify(ZEOTAP_COOKIE)),
+    (new Date(Date.now() + 5000).toUTCString()),
+  );
+}
+
+function unsetCookie() {
+  storage.setCookie(ZEOTAP_COOKIE_NAME, '');
+}
+
+function unsetLocalStorage() {
+  storage.setDataInLocalStorage(ZEOTAP_COOKIE_NAME, '');
+}
+
+describe('Zeotap ID System', function() {
+  describe('test method: getId', function() {
+    afterEach(() => {
+      unsetCookie();
+      unsetLocalStorage();
+    })
+
+    it('return undefined if incorrect config as argument', function() {
+      writeZeotapCookie();
+      let id = zeotapIdPlusSubmodule.getId({});
+      expect(id).to.be.undefined;
+    });
+
+    it('provides the stored Zeotap id if a cookie exists', function() {
+      writeZeotapCookie();
+      let id = zeotapIdPlusSubmodule.getId(CONFIG_PARAMS_MOCK);
+      expect(id).to.deep.equal({
+        id: ZEOTAP_COOKIE
+      });
+    });
+
+    it('provides the stored Zeotap id if cookie is absent but present in local storage', function() {
```

Comment:
Thanks for pointing this out @smenzer , the purpose of this test was to check whether zeotap id is returned if id is present in local storage instead of cookies.
Have made changes to the test.